butterknife-runtime/src/main/java/butterknife/internal/DebouncingOnClickListener.java

```diff
@@ -13,8 +15,14 @@
 
   @Override public final void onClick(View v) {
     if (enabled) {
-      enabled = false;
-      v.post(ENABLE_AGAIN);
```

Comment:
We can switch this to post to the main looper directly rather than going through the view. That way we don't need to check for attach.

Reply:
Thanks for your reply.

Yes, i updated my code and pushed to my fork. But the Travis CI build failed, i don't known why...

CI failed log:

```
Task :butterknife-integration-test:connectedCodegenDebugAndroidTest FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':butterknife-integration-test:connectedCodegenDebugAndroidTest'.
> There were failing tests. See the report at: file:///home/travis/build/JakeWharton/butterknife/butterknife-integration-test/build/reports/androidTests/connected/flavors/CODEGEN/index.html
```

Reply:
Hi Jake, I fixed unit test errors and The Travis CI build passed😀.